client/orm/qb/select.go

```diff
@@ -244,3 +244,18 @@ func (s *Selector[T]) Get(ctx context.Context) (*T, error) {
 	err = s.db.ReadRaw(ctx, t, q.SQL, q.Args...)
 	return t, nil
 }
+
+func (s *Selector[T]) WhereMap(conds map[string]any) *Selector[T] {
+	psArr := make([]Predicate, 0, len(conds))
+	for k, v := range conds {
+		ps := C(k).EQ(v)
+		psArr = append(psArr, ps)
+	}
+	return s.Where(psArr...)
```

Comment:
Please help to make the change:
```go
func (s *Selector[T]) Where(ps ...Predicate) *Selector[T] {
	s.where = ps // s.where = append(s.where, ps)
	return s
}
```

Reply:
I think this change makes the Where more convenient. And please help to add test for this change.

Reply:
After making this change, you can change to:
```go
	for k, v := range conds {
		ps := C(k).EQ(v)
		s.Where(ps) // avoid memory allocation
	}
```

Reply:
I solved the above problem. Add  test example for WhereRaw and WhereMap and Where, but find that WhereMap test might fail because the traversal of the map is unordered.

Reply:
use one key-value pair. It's a simple function